### DISCUSSION TITLE
How can a beginner in C can start contributing to zephyr?

### DISCUSSION THREAD

**Discussion by adirsingh96** [zephyrproject-rtos/zephyr#46731] [General] [answered]
Hello everyone, I have been learning C from past few weeks and wanted to work on some real world project, How can I start contributing to Zephyr?

**Comment by yashi** [comment, marked as answer]
There is a label called ["Good first issue"](https://github.com/zephyrproject-rtos/zephyr/labels/Good%20first%20issue), which you might want to take a look at.

Another idea is to start with documentation fixes.  Just read [Getting Started Guide](https://docs.zephyrproject.org/latest/develop/getting_started/index.html) or any doc we have and fix bugs you find while reading.  I know you want to work on C but learning Github and the build process might help at first.

**Reply by adirsingh96** [reply]
Thanks a lot, will check out the document and learn about using GitHub first.
